custom/icds_reports/ucr/data_sources/ls_vhnd_form.json

```diff
@@ -0,0 +1,134 @@
+{
+  "domains": [
+    "icds-dashboard-qa",
+    "reach-sandbox",
+    "reach-test",
+    "icds-sql",
+    "icds-test",
+    "icds-cas",
+    "cas-lab",
+    "icds-cas-sandbox"
+  ],
+  "server_environment": [
+    "softlayer",
+    "icds-new",
+    "icds"
+  ],
+  "config": {
+    "table_id": "static-ls_vhnd_form",
+    "display_name": "Forms - LS VHND (Static)",
+    "referenced_doc_type": "XFormInstance",
+    "description": "",
+    "base_item_expression": {},
+    "configured_filter": {
+      "operator": "in",
+      "expression": {
+        "datatype": null,
+        "type": "property_name",
+        "property_name": "xmlns"
+      },
+      "type": "boolean_expression",
+      "property_value": [
+        "http://openrosa.org/formdesigner/b8273b657bb097eb6ba822663b7191ff6bc276ff"
+      ]
+    },
+    "configured_indicators": [
+      {
+        "display_name": "Supervisor ID",
+        "datatype": "string",
+        "type": "expression",
+        "transform": {},
+        "is_nullable": true,
+        "is_primary_key": false,
+        "column_id": "location_id",
+        "expression": {
+          "type": "named",
+          "name": "user_location_id"
+        }
+      },
+      {
+        "display_name": null,
+        "datatype": "date",
+        "expression": {
+          "datatype": "date",
+          "type": "property_path",
+          "property_path": [
+            "form",
+            "vhnd_date"
+          ]
+        },
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "type": "expression",
+        "column_id": "vhnd_date"
+      },
+      {
+        "display_name": null,
+        "transform": {},
+        "datatype": "date",
+        "expression": {
+          "expression": {
+            "datatype": "date",
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "meta",
+              "timeEnd"
+            ]
+          },
+          "type": "root_doc"
+        },
+        "is_primary_key": false,
+        "is_nullable": true,
+        "column_id": "submitted_on",
+        "type": "expression"
+      },
+      {
+        "display_name": null,
+        "transform": {},
+        "datatype": "date",
+        "is_primary_key": false,
+        "create_index": true,
+        "is_nullable": true,
+        "column_id": "month",
```

Comment:
Why is this column needed? It should be able to be derived from the `submitted_on` column

Reply:
you mean it can be derived from `submitted_on` directly during sql query while aggregation?